### ISSUE TITLE
Misaligned Image for Application Movement on macOS

### ISSUE THREAD

**Issue by H7USHKA** [Veil-Project/veil#34]
Mojave 10.14.2

<img width="500" alt="screen shot 2018-12-26 at 11 04 48 pm" src="https://user-images.githubusercontent.com/23407761/50487748-510bc580-09b4-11e9-8c40-dcd4974255f8.png">

**Comment by seanPhill** [comment]
I know this is low priority, but I found the AppleScript fail that causes this! ;)
In "contrib/macdeploy/custom_dsstore.py" on line 56 "Bitcoin-Qt.app" would need to match "Veil Core.app" in order to match what is actually in the DMG image for the AppleScript that aligns the icons. Another contributing factor might be that this should be named by a variable such as 'CFBundleDisplayName' by the looks of it, and other code suggests it was intended to produce the app as "Veil-Qt.app" rather than "Veil Core.app". Also, once this is changed the copyright above can be changed. I tried to do this myself, but ended up creating multiple commits, so did not want to submit a pull request for the mess, which didn't take into account the possible issue about the app name.
